src/Utilities/ErrorHandling/AbortWithErrorMessage.hpp

```diff
@@ -9,14 +9,21 @@
 #include <string>
 
 /// \ingroup ErrorHandlingGroup
-/// Compose an error message with an expression and abort the program.
+/// Compose an error message with an expression and a back trace, then abort the
+/// program.
 [[noreturn]] void abort_with_error_message(const char* expression,
                                            const char* file, int line,
                                            const char* pretty_function,
                                            const std::string& message);
 
 /// \ingroup ErrorHandlingGroup
-/// Compose an error message and abort the program.
+/// Compose an error message including a back trace and abort the program.
```

Comment:
some dox strings call it a `back trace`, others say `backtrace` - best to be consistent.